js/models/wallet/Spend.js

```diff
@@ -106,6 +114,14 @@ class Spend extends BaseModel {
         if (attrs.memo && typeof attrs.memo !== 'string') {
           addError('memo', 'If provided, the memo should be a string.');
         }
+
+        if (
+          this.url === getOrderSpendUrl() &&
+          typeof attrs.orderId !== 'string' ||
+          !attrs.orderId
```

Comment:
This error is firing when trying to send funds directly from the wallet. 

I think you want the second two conditions in parenthesis:
```
this.url === getOrderSpendUrl() &&
          (typeof attrs.orderId !== 'string' ||
          !attrs.orderId)
```